docs/help/operator.md

```diff
@@ -844,9 +844,41 @@ spec:
 ## Providing additional brokerProperties configuration from a secret
 It is possible to replace the use of the activemqartemisaddresses CRD and much of the activemqartemissecurities CRD with configuration via broker properties. This can necessitate a large amount of configuration in the CR.brokerProperties field.
 In order to provide a way to split or orgainse these properties by file or by secret, an extra mount can be used to provide a secret that will be treated as an additional source of broker properties configuration.
+
 Using an **extraMounts** secret with a suffix "-bp" will cause the operator to auto mount the secret and make the broker aware of it's location. In addition the CR.Status.Condition[BrokerPropertiesApplied] will reflect the content of this secret.
+
 Broker properties are applied in order, starting with the CR.brokerProperties and then with the "-bp" auto mounts in turn. Keys (or property files) from secrets are applied in alphabetical order.
 
+To configure a specific broker instance in a "-bp" secret, use `broker-N` as the prefix for a key in the secret data. For example:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: config-1-bp
+stringData:
+  journal1.properties: |
+    journalFileSize=12345
+  broker-0.first.properties: |
+    globalMaxSize=512m
+  broker-1.first.properties: |
```

Comment:
in the test there is first and second. I think they can both be called the same name, but it may be worth being consistent...

also, maybe call it: globalMemory.properties - to indicate that the config maps can help to categorise the config.

Reply:
+1